.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: Continuous Integration
+
+on:
+  push:
+
+jobs:
+  test:
+    name: Unit Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-go@v5
+        with:
+          go-version: 1.19
```

Comment:
Can it be updated to 1.22? 1.19 is outside of the support window

Reply:
Created another PR (to develop) which I plan to merge before this one.